packages/runtime/container-runtime/src/test/opLifecycle/remoteMessageProcessor.spec.ts

```diff
@@ -172,7 +173,8 @@ describe("RemoteMessageProcessor", () => {
 					referenceSequenceNumber: message.referenceSequenceNumber,
 				} as ISequencedDocumentMessage;
 
-				const processResult = messageProcessor.process(inboundMessage);
+				ensureContentsDeserialized(inboundMessage, true, () => {});
```

Comment:
@dannimad I have a slight preference for this test code being more explicit, rather than using the `ensureContentsDeserialized` helper which is not opinionated about the input.  i.e. `contents: JSON.parse(message.contents)` on when instantiating `inboundMessage`.

Reply:
I think I prefer this approach, as is a type of indirect test to `ensureContentsDeserialized` as well. I could change it anyway.